### ISSUE TITLE
Introduction to neural networks

### ISSUE THREAD

**Issue by antoniogamiz** [antoniogamiz/tfg#29]
We need to explain the basics of:

- Gradient descent
- Back propagation

These two concepts are vital for developing the word2vec model and its extension.

**Comment by JJ** [comment]
Issues have to be informed by a user story always. Software engineering is a problem-solving discipline.

**Comment by antoniogamiz** [comment]
This issue is inside milestone 1, which tries to solve the fourth user story, math (#12).

**Comment by JJ** [comment]
The user story has to be referenced in the body, not in a comment.
